simulation_parameters/SimulationParameters.cs

```diff
@@ -62,6 +63,8 @@ private SimulationParameters()
 
         helpTexts = LoadRegistry<HelpTexts>("res://simulation_parameters/common/help_texts.json");
 
+        inputGroups = LoadListRegistry<NamedInputGroup>(Constants.INPUT_OPTIONS);
```

Comment:
I'm guessing the other registry loads in `SimulationParameters` will be changed to also use constants for the path. If not, shouldn't this pass the string of the full path directly? (since the other loads do that)

Reply:
I just copied over the use of the constant. It's kinda redundant here as only the simulation paramters should ever need to know the path to the JSON files it loads. I just couldn't be bothered to think and change it as I'd already spent so much time on this PR.

Reply:
Inlined this string.